mm2src/coins/eth/eth_tests.rs

```diff
@@ -153,6 +154,7 @@ fn eth_coin_from_keypair(
         logs_block_range: DEFAULT_LOGS_BLOCK_RANGE,
         nonce_lock: new_nonce_lock(),
         erc20_tokens_infos: Default::default(),
+        non_fungible_tokens_infos: Default::default(),
```

Comment:
is there a reason why you don't want to use  `nft_.. ` instead of `non_fungible_tokens...`

Reply:
Good question! I mentioned it on the call, but I got that without a context it was difficult for the team to understand what I was talking about. 

I dont like `nft_token` name, because it is like `non_fungible_token_token`. Word repetition hurts me a little :grimacing: 
Well I used `nft_token` format when big naming wasnt appropriate like params or values names 
![image](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/d2ebd63c-3459-4104-9cca-ec7a2f7dd164)
 
But on the other hand I understand, its kinda a rule in the repo to form names like `..._token`.

However, using full naming `non_fungible_token` could mean that this is a part of big feature like Swaps/Coins behavior.
But I dont insist, if the team decide that we should  stick to the `nft_tokens` name, I will do the changes

@shamardy sorry to ping you, its not urgent just small naming issue/question

Reply:
> IMHO, it can either be `nfts_infos` or `non_fungible_tokens_infos`.

Oh exactly, I can just skip `tokens` word -_-' 
Changed long filed/function names to `nfts_infos`. Short name is better.